ci/shell.nix

```
@@ -39,6 +39,8 @@ let
   };
   gcc = nixpkgs.gcc11;
   llvmPackages = nixpkgs.llvmPackages_12;
+  # see pyright/README.md for update procedure
+  pyright = nixpkgs.callPackage ./pyright/default.nix {};
```

small nit-pick - `default.nix` is not required because it is default

```suggestion
  pyright = nixpkgs.callPackage ./pyright {};
```